frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -40,14 +42,27 @@ const InfiniteProjectOverview = () => {
     const { project, loading: projectLoading } = useProject(projectId, {
         refreshInterval,
     });
-    const [nextCursor, setNextCursor] = useState('');
+    const [cursor, setCursor] = useState('');
     const {
         features: searchFeatures,
+        nextCursor,
+        total,
         refetch,
         loading,
-    } = useFeatureSearch(nextCursor, projectId, { refreshInterval });
+    } = useFeatureSearch(cursor, projectId, { refreshInterval });
+
     const { members, features, health, description, environments, stats } =
         project;
+    const fetchNextPageRef = useOnVisible<HTMLDivElement>(() => {
```

Comment:
when the load more marker is reached we update the cursor. We also need to prevent multiple requests so checking for the cursor change